deploy/sre-prometheus/ocm-agent/unsupported-logging/100-ocm-agent.PrometheusRule.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-managed-notification-alerts
+    role: alert-rules 
+  name: elasticsearch-sre-managed-notification-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-managed-notification-alerts
```

This should probably be a distinct group
```suggestion
  - name: sre-elasticsearch-managed-notification-alerts
```